docs/user_guide.md

```diff
@@ -252,6 +256,95 @@ The images can be filtered by labels:
+
+If CVEs are added to the system whitelist after you have created a project whitelist, click **Add System** to add the new entries from the system whitelist to the project whitelist. 
+
+**NOTE**: If CVEs are deleted from the system whitelist after you have created a project whitelist, and if you added the system whitelist to the project whitelist, you must manually remove the deleted CVEs from the project whitelist. If you click **Add System** after CVEs have been deleted from the system whitelist, the deleted CVEs are not automatically removed from the project whitelist.
+
+## Set Project Quotas
+
+To exercise control over resource use, as a system administrator you can set  quotas on projects. You can limit the number of artifacts that a project can contain and limit the amount of storage capacity that a project can consume. You can set default quotas that apply to all projects globally. You can also set quotas on individual projects. If you set global default quotas and also set different quotas on individual projects, the per-project quotas override the global default quotas.
+
+By default, all projects have unlimited quotas for both artifacts and storage use. 
+
+1. Go to **Configuration** > **Project Quotas**.
+   ![Project quotas](img/project-quota1.png)
+1. To set global default quotas on all projects, click **Edit**.
+   ![Project quotas](img/project-quota2.png)
+   1. For **Default artifact count**, enter the maximum number of artifacts that any project can contain. 
+   
+   Enter `-1` to set the default to unlimited. 
+   1. For **Default storage consumption**, enter the maximum quantity of storage that any project can consume, selecting `MB`, `GB`, or `TB` from the drop-down menu. 
+   
+   Enter `-1` to set the default to unlimited.
+   ![Project quotas](img/project-quota3.png)
+   1. Click **OK**.
+1. To set quotas on an individual project, click the 3 vertical dots next to a project and select **Edit**.
+   ![Project quotas](img/project-quota4.png)
+   1. For **Default artifact count**, enter the maximum number of artifacts that this individual project can contain, or enter `-1` to set the default to unlimited. 
+   1. For **Default storage consumption**, enter the maximum quantity of storage that this individual project can consume, selecting `MB`, `GB`, or `TB` from the drop-down menu.
+
+After you set quotas, the you can see how much of their quotas each project has consumed in the **Project Quotas** tab.
+
+![Project quotas](img/project-quota5.png)
+
+### How Harbor Calculates Resource Usage
+
+When setting project quotas, it is useful to know how Harbor calculates artifact numbers and storage use, especially in relation to image pushing, retagging, and garbage collection.
+
+- Harbor computes image size when blobs and manifests are pushed from the Docker client.
+- Harbor computes artifact counts when manifests are pushed from the Docker client.

```

Comment:
There's a salient point here which is manifest is pushed at the end after all blobs have been pushed successfully so in the case of pushing 2 images at once when there's only one 'spot' left, they don't get rejected right away which obviates a lot of the unnecessary pushing of blobs but only after the faster of the two is pushed.  Not sure if worth mentioning. @stuclem

Reply:
@xaleeks it's your call. If you think it's necessary, I'll add it. I suppose the question is "will customers notice and be confused about why one of their images didn't push?"

Reply:
Hey Stuart, if you can make an attempt at wording this in a manner which answers the question you pointed out, would be appreciated.  @stuclem

Reply:
@xaleeks how's this?

**NOTE**: When users push an image, the manifest is pushed last, after all of the associated blobs have been pushed successfully to the registry. If several images are pushed concurrently and if there is an insufficient number of artifacts left in the quota for all of them, images are accepted in the order that their manifests arrive. Consequently, an attempt to push an image might not be immediately rejected for exceeding the quota. This is because there was availability in the artifact quota when the push was initiated, but by the time the manifest arrived the quota had been exhausted.

Reply:
perfect! Thx @stuclem